sympy/physics/control/lti.py

```diff
@@ -0,0 +1,1033 @@
+from sympy import (
+    Symbol, simplify, oo, exp,
+    Poly, lcm, LC, degree, Integral, integrate,
+    Matrix, eye, zeros, Basic, latex,
+    ShapeError, ImmutableMatrix, MutableMatrix,
+    SparseMatrix, MutableDenseMatrix
+)
+
+from sympy.printing import sstr
+
+# import mpmath for numercial results
+from mpmath import expm, quad, matrix as mpm_matrix
+
+__all__ = ['StateSpaceModel', 'TransferFunctionModel']
+
+_matrixTypes = (
+    Matrix, ImmutableMatrix, MutableMatrix, SparseMatrix, MutableDenseMatrix)
+
+
+class StateSpaceModel(Basic):
+    """State space model (ssm) of a linear, time invariant control system
+
+    Represents the standard state-space model with state matrix A, input matrix B, output matrix C, and
+    transmission matrix D. This makes the linear control system:
+        (1) x'(t) = A * x(t) + B * u(t);    x in R^n , u in R^k
+        (2) y(t)  = C * x(t) + D * u(t);    y in R^m
+    where u(t)  is any input signal, y(t) the corresponding output, and x(t) the system's state.
+
+    Parameters
+    ==========
+
+    *arg : TransferFunctionModel, Sympy-Matrices
+        tfm to construct the state space model from, or the Matrices A, B, C, D as StateSpaceModel(A, B, C, D)
```

Comment:
These lines seem a bit longer that 79 characters.

Reply:
Thanks, I will follow PEP 8.